### ISSUE TITLE
Add custom place to Nearby

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#2020]
Sometimes I hear about an interesting non-famous place on the local news.
Because it is very local, this place has no Commons picture and of course no Wikidata item.
But still, I would like to have a way to remember it in the future.

Nearby seems like a good place to put such information. I should be able to long-press anywhere on the map and add the new point's label.

There are many ways to implement backend storage:
- Locally in the app's database. Easy, but points get lost at each reinstall
- On the SD card on the phone. A bit awkward and lost when people change phones.
- As a list in a sub-page of the user. In such case we should be clear that these points are publicly visible and linked to the user.
- As a global list shared by all users of the app. Same warning as above.

In any case, loading should happen in parallel with Wikidata SPARQL loading, and points should start showing up as soon as the first points start arriving.

Low priority.

**Comment by Anil-Praharaj21** [comment]
I am working

**Comment by maskaravivek** [comment]
@Anil-Praharaj21 can you share your approach here? :) @nicolas-raoul suggested 4 ways to implement the feature and it would be great if we could first reach a consensus on which method we should go with.

**Comment by Anil-Praharaj21** [comment]
I am thinking to save it in the database or we can go for Bookmarked it
